src/server/validateDependencies.ts

```diff
@@ -41,8 +41,65 @@ const DL_OPEN_LIBRARIES = {
 
 async function validateDependencies(browserPath: string, browser: BrowserDescriptor) {
   // We currently only support Linux.
-  if (os.platform() !== 'linux')
+  if (os.platform() === 'linux')
+    return await validateDependenciesLinux(browserPath, browser);
+  if (os.platform() === 'win32' && os.arch() === 'x64')
+    return await validateDependenciesWindows(browserPath, browser);
+}
+
+async function validateDependenciesWindows(browserPath: string, browser: BrowserDescriptor) {
+  const directoryPaths = windowsExeAndDllDirectories(browserPath, browser);
+  const lddPaths: string[] = [];
+  for (const directoryPath of directoryPaths)
+    lddPaths.push(...(await executablesOrSharedLibraries(directoryPath)));
+  const allMissingDeps = await Promise.all(lddPaths.map(lddPath => missingFileDependenciesWindows(lddPath)));
+  const missingDeps: Set<string> = new Set();
+  for (const deps of allMissingDeps) {
+    for (const dep of deps)
+      missingDeps.add(dep);
+  }
+
+  if (!missingDeps.size)
     return;
+
+  let knownIssues = [];
+  for (const dep of missingDeps) {
+    if (dep.startsWith('api-ms-win-crt-')) {
+      knownIssues.push([
+        `Some of the Universal C Runtime files cannot be found on the system. You can fix`,
+        `that by installing Microsoft Visual C++ Redistributable for Visual Studio from:`,
+        `https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads`,
+        ``,
+      ].join('\n'));
+      break;
```

Comment:
why breaking and doing another loop down below? Instead, you can do

```js
const knownIssues = new Set(); // to avoid dupes!
if (dep.startsWith('api-ms-win-crt-')) {
  // ....
  knownIssues.add('one known issue');
} else if (dep === "mf.dll" || dep === "mfplat.dll" ||  dep === "msmpeg2vdec.dll") {
  // ...
  knownIssues.add('another known issue');
}
```